src/molecules/radio-input/radio-input.scss

```diff
@@ -1,5 +1,5 @@
 @import "../../assets/scss/6-components/_component-base";
-@import "../buttons/button.scss";
+@import "../../Atoms/buttons/button.scss"; // Todo: Remove this dependency
```

Comment:
Inconsistent capitalization here

Reply:
#86 - All files related to a component should live in its folder. Components that are dependent on another component's styles can create undesired and unexpected problems when stuff gets updated. A better solution would be to have each component's styles self-contained within it's own style sheet and if there are any shared styles, those things should be broken out into a mixin, function, etc.

I removed those comments for now (I hadn't intended them to get committed anyway, sorry!) and we can pick this up in a different PR.